test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/DefaultObjectValidatorTests.cs

```diff
@@ -1066,6 +1068,33 @@ public void Validate_ForExcludedCollectionType_PropertiesMarkedAsSkipped()
             Assert.Empty(entry.Errors);
         }
 
+        [Fact]
+        public void Validate_SuppressesValidation_ForExcludedType_Stream()
+        {
+            // Arrange
+            var options = new MvcOptions();
+            var optionsSeteup = new MvcCoreMvcOptionsSetup(Mock.Of<IHttpRequestStreamReaderFactory>());
+            optionsSeteup.Configure(options);
+            var validator = CreateValidator(providers: options.ModelMetadataDetailsProviders.ToArray());
+            var model = new MemoryStream(Encoding.UTF8.GetBytes("Hello!"));
+            var actionContext = new ActionContext();
+            var modelState = actionContext.ModelState;
+            modelState.SetModelValue("parameter", rawValue: null, attemptedValue: null);
+            var validationState = new ValidationStateDictionary();
+            validationState.Add(model, new ValidationStateEntry() { Key = "parameter" });
+
+            // Act
+            validator.Validate(actionContext, validationState, "parameter", model);
+
+            // Assert
+            Assert.True(modelState.IsValid);
+            AssertKeysEqual(modelState, "parameter");
+
+            var entry = modelState["parameter"];
```

Comment:
Simplify these lines to avoid the `AssertKeysEqual()` "helper":
``` c#
var entry = Assert.Single(modelState);
```
(You don't need to check the dictionary key because you're manually adding the `ModelStateEntry` in `// Arrange`.)